cogs/util/bot.py

```diff
@@ -133,6 +133,8 @@ def _check_table_exists(self, tablename):
                 except discord.Forbidden:
                     # we can't send messages in that channel
                     return
+            elif isinstance(exception.original, discord.HTTPException) and exception.original.status == 400:
+                return await ctx.send('Congratulations! I can\'t send that message.')
```

Comment:
you might also want to wrap this in a `try: ... except: pass` just in case sending fails here too

Reply:
Yep good idea. Just added that.